### ISSUE TITLE
Change volume audio output

### ISSUE THREAD

**Issue by bachhoan88** [sannies/mp4parser#208]
I want set volume audio when export video using `Track.getTrackMetaData().setVolume(0.0f)`
But video export still has audio. Can you help me to set volume video output please

**Comment by quangtd95** [comment]
have you find the way to solve this problem?

**Comment by bachhoan88** [comment]
Yes, Because my application using two audio (audio from video + audio from mp3 song) should be use ffmpeg for set audio video output

**Comment by quangtd95** [comment]
My application is running on Android platform, and ffmpeg is very slow. So I use mp4parser to adjust volume but it doesn't work!

**Comment by bachhoan88** [comment]
Correctly ffmpeg is very slow, but Y can just use which for update audio, It very fast (video input 30s volume 50%, audio input 1'30s - volume 50% ==> Export video: 4-5s) 
If you want to a ffmpeg code to change volume, I'll send it for y

**Comment by quangtd95** [comment]
please send me your code. my email: quang.td95@gmail.com 
thank you very much :)

**Comment by Sdghasemi** [comment]
I'm also wondering how to change the volume level of the output video as a float between `0.0f` and `1.0f` or even `2.0f` maybe (boost audio level) with mp4parser, I can't use ffmpeg for some reasons. It would be a great help if someone provide me a solution to do so.
Thanks in advance

**Comment by buptlinyuan** [comment]
> Correctly ffmpeg is very slow, but Y should use that for update audio, It very fast (video input 30s volume 50%, audio input 1'30s - volume 50% ==> Export video: 4-5s)
> If you want to a ffmpeg code to change volume, I'll send it for y

hi, I have met the same problem, could you send it to me , my email : buptlinyuan@163.com , Thanks! @bachhoan88
